src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -36,14 +36,13 @@ case $OSName in
         ;;
 
     Linux)
-        source /etc/os-release
-        if [ "$ID" == "centos" ]; then
+        if [ "$(cat /etc/*-release | grep -cim1 centos)" -eq 1 ]; then
             __PUBLISH_RID=centos.7-x64
-        elif [ "$ID" == "rhel" ]; then
+        elif [ "$(cat /etc/*-release | grep -cim1 rhel)" -eq 1 ]; then
```

Comment:
Oh, wow, you introduced a wildcard here... that doesn't seem like "removing calls to `source`", but rather "lets do something different"